### ISSUE TITLE
Disabling features does not work

### ISSUE THREAD

**Issue by georgescubogdan** [ranbuch/accessibility#56]
Hi,

I tried disabling increaseLineHeight and decreaseLineHeight as per the available documentation and it doesn't seem to work.
Also, the interface IAccessibilityModulesOptions doesn't have those properties. 

I coudn't disable any of the functions.

Scenario: Anuglar 8.2.5, TypeScript 3.5.3, accessibility 4.5.7

Anyway, great tool. Really useful :)

**Comment by rudachenkoev** [comment]
Same problem with modules disabling
